wiki/About/ja.md

```diff
@@ -0,0 +1,6 @@
+osu!wikiについて
```

Comment:
idk if `osu!wiki` is needed or not :shrug:

Reply:
If you translate "About" to Japanese, it will be "について".
However, Japanese can not understand the meaning only with "について".
So, need to put some word in front.

Reply:
alright. btw, you may need to pm megaapple_pi in discord to help you squash your commits.